server/sonar-web/src/main/js/components/issue/issue-view.js

```diff
@@ -151,7 +151,7 @@ export default Marionette.ItemView.extend({
       this.disableControls();
       return $.ajax({
         type: 'POST',
-        url: '/api/issues/delete_comment?key=' + commentKey
+        url: window.baseUrl + '/api/issues/delete_comment?key=' + commentKey
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Convert this concatenation to the use of a template. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=javascript%3AS3512)